base/Dockerfile

```diff
@@ -14,6 +14,6 @@ RUN dnf -y update && \
         tar \
         util-linux \
         && \
-    dnf clean all
-RUN groupadd -g 1111 -r user && useradd -r -g user -u 1111 user
-RUN mkdir -p /home/user/.ssh && mkdir -p /usr/local/bin
+    dnf clean all && \
+    groupadd -g 1111 -r user && useradd -r -g user -u 1111 user && \
+    mkdir -p /home/user/.ssh && mkdir -p /usr/local/bin
```

Comment:
What's the rationale for this? Chaining the groupadd and mkdir stuff is fine, but IMHO it makes sense to keep the "quick & small" setup bits in a separate RUN command (and thus layer). That way, it's much faster to iterate and build a new container when changing those, as the docker layer caching can re-use the expensive "install package" step.